installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -0,0 +1,116 @@
+[id="installing-gcp-user-infra-vpc"]
+= Installing a cluster on GCP using Deployment Manager templates
+include::modules/common-attributes.adoc[]
+:context: installing-gcp-user-infra-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Google Cloud Platform (GCP) that uses infrastructure that you provide.
+
+The steps for performing a user-provided infrastructure install are outlined here. Several
+link:https://cloud.google.com/deployment-manager/docs[Deployment Manager] templates are provided to assist in
+completing these steps or to help model your own. You are also free to create
+the required resources through other methods; the templates are just an
+example.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* You must have an existing VPC in GCP that contains separate subnets for the compute and control plane machines that the cluster requires.
+* If you use a firewall and plan to use telemetry, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure the firewall to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+[id="csr-management-gcp-vpc"]
+== Certificate signing requests management
+
+Because your cluster has limited access to automatic machine management when you
+use infrastructure that you provision, you must provide a mechanism for approving
+cluster certificate signing requests (CSRs) after installation. The
+`kube-controller-manager` only approves the kubelet client CSRs. The
+`machine-approver` cannot guarantee the validity of a serving certificate
+that is requested by using kubelet credentials because it cannot confirm that
+the correct machine issued the request. You must determine and implement a
+method of verifying the validity of the kubelet serving certificate requests
+and approving them.
+
+[id="installation-gcp-user-infra-config-project-vpc"]
+== Configuring your GCP project
+
+Before you can install {product-title}, you must configure a Google Cloud
+Platform (GCP) project to host it.
+
+include::modules/installation-gcp-project.adoc[leveloffset=+2]
+include::modules/installation-gcp-enabling-api-services.adoc[leveloffset=+2]
+include::modules/installation-gcp-dns.adoc[leveloffset=+2]
```

Comment:
In shared VPC scenario, host project hosts the DNS zone for the cluster which means DNS is in the different project with OCP cluster. Please update the content in this section accordingly.